### DISCUSSION TITLE
Any plans for Quarkus support

### DISCUSSION THREAD

**Discussion by eijckron** [kagkarlsson/db-scheduler#499] [Q&A]
Are there any plans for a db-scheduler Quarkus extension?
If not are there Java EE examples that could be used as a basis to integrate db-scheduler in Quarkus

**Comment by gustavkarlssonft** [comment]
Quarkus-support would have to be contributed by someone familiar with it. Nothing on the roadmap.

It is easy to use the `SchedulerBuilder` to instantiate and configure the scheduler in Quarkus
